classes/02class/exercises/c04-iam01/nimmi89/README.md

```diff
@@ -0,0 +1,92 @@
+# c04-iam01
+
+## Command Execution Output
+- [cli_commands.txt](cli_commands.txt)
+```
+    >>Command to create new user
+    aws iam create-user  --user-name user_readonly_S3
+    >The command returns:
+    {
+        "User": {
+            "Path": "/",
+            "UserName": "user_readonly_S3",
+            "UserId": "AIDAWMG45LNQQK3F2RZDK",
+            "Arn": "arn:aws:iam::438549961569:user/user_readonly_S3",
+            "CreateDate": "2020-07-02T08:26:34+00:00"
+        }
+    }
+    >Policy file with required permissions
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:Get*",
+                    "s3:List*"
+                ],
+                "Resource": [
+                    "*"
+                ]
+            }
+        ]
+    }
+
+    >Command to create a policy 
```

Comment:
```suggestion
## Command to create new user
```
    aws iam create-user  --user-name user_readonly_S3
    >The command returns:
    {
        "User": {
            "Path": "/",
            "UserName": "user_readonly_S3",
            "UserId": "AIDAWMG45LNQQK3F2RZDK",
            "Arn": "arn:aws:iam::438549961569:user/user_readonly_S3",
            "CreateDate": "2020-07-02T08:26:34+00:00"
        }
    }
```

## Policy file with required permissions

```
    {
        "Version": "2012-10-17",
        "Statement": [
            {
                "Effect": "Allow",
                "Action": [
                    "s3:Get*",
                    "s3:List*"
                ],
                "Resource": [
                    "*"
                ]
            }
        ]
    }
```

## Command to create a policy 


```